misc/history/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+const path = require("path");
+
+module.exports = {
+  mode: "production",
+  entry: {
+    index: path.resolve(__dirname, "dist", "cjs", "index.js"),
+  },
+  output: {
+    path: path.resolve(__dirname, "dist", "umd"),
+    filename: "[name].min.js",
+    libraryTarget: "umd",
+    library: "history",
```

Comment:
This feels a bit too generic for a UMD build name, maybe:

```suggestion
    library: "walletConnectHistoryUtil",
```